packages/devextreme/js/__internal/scheduler/appointment_popup/m_popup.ts

```diff
@@ -438,20 +442,21 @@ export class AppointmentPopup {
             this._saveButtonClickHandler(e);
           },
         },
-      },
-      {
-        toolbar: 'top',
-        location: 'after',
-        widget: 'dxButton',
-        options: {
-          text: messageLocalization.format('Cancel'),
-          stylingMode: 'outlined',
-          onClick: (): void => {
-            this.hide();
-          },
+      });
+    }
+
+    toolbarItems.push({
+      toolbar: 'top',
+      location: 'after',
+      widget: 'dxButton',
+      options: {
+        text: messageLocalization.format('Cancel'),
+        stylingMode: 'outlined',
+        onClick: (): void => {
+          this.hide();
```

Comment:
I have noticed that for the toolbar of the main form `shortcut` property is used for  'cancel' and 'save' buttons ([here](https://github.com/sjbur/DevExtreme/blob/fe2172fec444363a43e597b8b43745ec6c32ef31/packages/devextreme/js/__internal/scheduler/appointment_popup/m_popup.ts#L499)).

I would like to ask to check we can use 'shortcut' for 'cancel' and 'save' for recurrence toolbar buttons too. Thus we could remove some properties from 'cancel' and 'save' buttons config